catalog/core/catalog-core-urlresourcereader/src/main/java/ddf/catalog/resource/impl/URLResourceReader.java

```diff
@@ -465,6 +477,23 @@ private void skipBytes(InputStream is, String bytesToSkip) throws IOException {
         }
     }
 
+    private void alignStream(InputStream in, long requestedBytesToSkip, long responseBytesSkipped)
+            throws IOException {
+        try {
+            if (requestedBytesToSkip > responseBytesSkipped) {
+                LOGGER.error(
+                        "Source incorrect byte range, skipping ahead [{}] bytes in the input stream",
+                        requestedBytesToSkip - responseBytesSkipped);
+                long skipped = in.skip(requestedBytesToSkip - responseBytesSkipped);
```

Comment:
You're not doing anything with the result of the `skip` call. Might as well not get it.